modules/registry-accessing-directly.adoc

```diff
@@ -7,54 +7,53 @@
 
 You can access the registry directly to invoke `podman` commands. This allows
 you to push images to or pull them from the integrated registry directly using
-operations like `podman-push` or `podman-pull`. To do so, you must be logged in
-to the registry using the `podman-login` command. The operations you can perform
+operations like `podman push` or `podman pull`. To do so, you must be logged in
+to the registry using the `podman login` command. The operations you can perform
 depend on your user permissions, as described in the following sections.
 
 .Prerequisites
 
-* For any direct access, you must have a regular user for your preferred identity
-provider.
-** A regular user can generate an access token required for logging in to
-the registry.
-** System users, such as *system:admin*, cannot obtain access tokens
-and, therefore, cannot access the registry directly.
-+
-For example, if you are using `HTPASSWD` authentication, you can create one
-using the following command:
-+
-----
-# htpasswd /etc/origin/openshift-htpasswd <user_name>
-----
-
-* For pulling images, for example when using the `podman-pull` command,
-the user must have the *registry-viewer* role. To add this role:
+* You must have cluster administrator privileges.
+* You must have configured an identity provider (IDP).
+* For pulling images, for example when using the `podman pull` command,
+the user must have the `registry-viewer` role. To add this role:
 +
 ----
 $ oc policy add-role-to-user registry-viewer <user_name>
 ----
 
-* For writing or pushing images, for example when using the `podman-push` command,
-the user must have the *registry-editor* role. To add this role:
+* For writing or pushing images, for example when using the `podman push` command,
+the user must have the `registry-editor` role. To add this role:
 +
 ----
 $ oc policy add-role-to-user registry-editor <user_name>
 ----
 
 .Procedure
 
-. Log in to the registry directly:
-
-.. Ensure you are logged in to {product-title} as a *regular user*:
+. Ensure you are logged in to {product-title} as a cluster administrator:
 +
 ----
 $ oc login
 ----
 
-.. Log in to the container image registry by using your access token:
+. Look up the route for the registry from the `openshift-image-registry`
+namespace:
++
+----
+$ oc get routes -n openshift-image-registry
+----
++
+[NOTE]
+====
+Since the Image Registry Operator creates the route, it will likely be the
+`default-route`.
```

Comment:
The default route address will be like: default-route-openshift-image-registry.<cluster_name>, for example default-route-openshift-image-registry.apps.qe-521.qe.devcluster.openshift.com